packages/react/src/DataTable/Table.tsx

```diff
@@ -394,15 +394,15 @@ function TableSortHeader({align, children, direction, onToggleSort, ...rest}: Ta
           <SortAscIcon
             className={clsx('TableSortIcon', 'TableSortIcon--ascending', {
               [classes.TableSortIcon]: enabled,
-              [classes['TableSortIcon--ascending']]: enabled,
+              [classes['TableSortIcon-ascending']]: enabled,
             })}
           />
         ) : null}
         {direction === SortDirection.DESC ? (
           <SortDescIcon
             className={clsx('TableSortIcon', 'TableSortIcon--descending', {
               [classes.TableSortIcon]: enabled,
-              [classes['TableSortIcon--descending']]: enabled,
```

Comment:
For this style I think it'd be helpful to preserve the original convention from BEM until we can update it to use our preferred approach with data attributes. Having `-descending` ends up as a weird hybrid between the two and I think it can lose the modifier meaning because of it.

Reply:
Makes sense, updated